tests/EStimLibrary.UnitTests/Core/SpatialModel/LocalizationTests.cs

```diff
@@ -0,0 +1,105 @@
+using EStimLibrary.Core.SpatialModel;
+using Xunit;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace EStimLibrary.UnitTests.Core.SpatialModel
```

Comment:
Style thing: 2 newlines between using and namespace declarations, then also put a ; at the end of the namespace declaration line so you can get rid of the outermost {} and unindent everything by one